### DISCUSSION TITLE
Images don’t show up

### DISCUSSION THREAD

**Discussion by 5ouma** [community/community#31424] [Mobile] [answered]
When the images on README.md named with **spaces** or **tags** _(#gh-light-mode-only)_, they don't show up.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.78.0 (76820214)
Device: iPhone 12 mini (iOS 15.6.1)
</details>

**Comment by toufiqul-hasan** [comment, marked as answer]
use camel case.
example: ghLightModeOnly

**Reply by 5ouma** [reply]
I changed them to camel case, but it didn't work.
Also, the tags lost their roles: both images(light and dark) showed up and were still hidden on mobile.

**Reply by toufiqul-hasan** [reply]
try this:
<*img src="./folderName/image.jpg" alt="image">

remove * from img tag

**Reply by 5ouma** [reply]
I've been using an HTML link type, but it's failed.

I think this is an issue with apps.

**Reply by toufiqul-hasan** [reply]
please share your code's screenshot

**Reply by 5ouma** [reply]
Here this is.

```html
<img width=500 src="path/to/image-light.png#gh-light-mode-only" alt="image-light">
<img width=500 src="path/to/image-dark.png#gh-dark-mode-only" alt="image-dark">
```

**Reply by toufiqul-hasan** [reply]
try this:
<*img width=500 src="path/to/image-light.png" alt="image-light">

remove * from img tag

**Reply by 5ouma** [reply]
Then that means I can't use the tags, right?
I want to change images according to appearance mode.

**Reply by toufiqul-hasan** [reply]
if you want to change images according to appearance mode then the image should be hosted on the github.

code example:

<*img width=500 src="https://user-images.githubusercontent.com/72879799/153904003-d7dee710-6552-4d23-a803-7a9a0ba67d92.png#gh-dark-mode-only" alt="image-dark">

remove * from img tag

**Reply by 5ouma** [reply]
Yes, those images are included in the repository.
Also, your code may be unable to change because there're no tags.

**Reply by toufiqul-hasan** [reply]
read this:
https://dev.to/arnavkr/using-darklight-mode-specific-images-in-github-markdown-3iln
I think this will help you to solve your problem.

**Reply by 5ouma** [reply]
Thank you for your reply!
Finally, I did it with the `https://user-images.githubusercontent.com/...` links.

I hope I can do this without creating links, just specifying images in the repository.

**Reply by CaptnCodr** [reply]
That's half my answer. 😅

**Reply by 5ouma** [reply]
I'm sorry, but my repository is private and can't show the links `https://raw.githubusercontent.com/...`.
However, `https://user-images.githubusercontent.com/...` links can be shown.

**Comment by CaptnCodr** [comment]
Link the images as raw hyperlink: `https://raw.githubusercontent.com/...`

**Reply by 5ouma** [reply]
My repository was private and they had to be with a token, so it couldn't be solved with your suggestion.
